s3repo/view.py

```diff
@@ -0,0 +1,26 @@
+from flask.views import MethodView
+from flask import render_template, Response
+import os
+
+class IndexView(MethodView):
```

Comment:
Why do you use inheritance from "MethodView"? Seems like the "View" class can be used here.

Reply:
I tried, as_view() needs "MethodView" heir.

Reply:
Sorry, but I don't understand you. Seems like the View class has the `as_view()` method (https://flask.palletsprojects.com/en/2.0.x/views/)

Reply:
Done, but I renamed View's get() to dispatch_request() as stated in the article.